arcgis-ios-sdk-samples/Route and directions/Navigate route with rerouting/NavigateRouteWithRerouting.storyboard

```diff
@@ -0,0 +1,99 @@
+        <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
+    </dependencies>
+    <scenes>
+        <!--Navigate Route With Rerouting View Controller-->
+        <scene sceneID="rti-Bq-cQX">
+            <objects>
+                <viewController id="QNZ-80-92R" customClass="NavigateRouteWithReroutingViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <view key="view" contentMode="scaleToFill" id="iHM-cv-Zt2">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="sd9-5N-pPZ" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="44" width="414" height="769"/>
+                            </view>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="cqm-kL-vVF">
+                                <rect key="frame" x="0.0" y="44" width="414" height="20.5"/>
+                                <subviews>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Directions are shown here." lineBreakMode="wordWrap" numberOfLines="4" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="EqP-Cf-t0c">
+                                        <rect key="frame" x="24" y="0.0" width="366" height="20.5"/>
+                                        <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                    </label>
+                                </subviews>
+                                <viewLayoutGuide key="safeArea" id="61W-JK-gvI"/>
+                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="0.59999999999999998" colorSpace="custom" customColorSpace="sRGB"/>
+                                <constraints>
+                                    <constraint firstItem="EqP-Cf-t0c" firstAttribute="leading" secondItem="61W-JK-gvI" secondAttribute="leading" constant="24" id="DRs-LJ-eY5"/>
+                                    <constraint firstItem="EqP-Cf-t0c" firstAttribute="trailing" secondItem="61W-JK-gvI" secondAttribute="trailing" constant="-24" id="EbJ-Dy-L7z"/>
```

Comment:
The leading and trailing constraints should be to the superview's margins, not the safe area, with a constant of `0`. Then, enable "Preserve Superview Margins" on the label's superview. That way, the expansion button, the label's text, and the Navigate button all line up nicely.

![image](https://user-images.githubusercontent.com/2257493/98307812-894e9880-1f7b-11eb-9731-d901178b0b49.png)

Reply:
![Simulator Screen Shot - iPhone 12 Pro Max - 2020-11-06 at 16 25 01](https://user-images.githubusercontent.com/52215269/98426181-bae36480-204c-11eb-8165-7711fd890b91.png)

This is what it looks like after I applied your suggestion. I originally added the constants because I felt that the words were too close to the edge. What do you think?

Reply:
You missed a key part of my suggestion: "The leading and trailing constraints should be to the superview's **margins**":

![image](https://user-images.githubusercontent.com/2257493/98432772-86849e00-2076-11eb-843d-f301676a9346.png)

With the leading and trailing edges of the label constrained to the superview's margins, the text is inset properly:

![image](https://user-images.githubusercontent.com/2257493/98432825-ff83f580-2076-11eb-8e9b-90a9990d27f2.png)